### PR TITLE
Use the CVO to manage operator config

### PR DESCRIPTION
**Pull request by damemi**
Don't use the EnsureOperatorConfigExists helper to create operator resource, instead tag the resource to be managed by the CVO

### CODE REVIEW

**Comment by damemi** [comment]
Opening this to get in line with our other operators before this helper is removed, but I'm not sure why it seems to be failing bindata. Running make update-bindata locally shows no changes for me.

I also see that you have the custom resource under `hack/` here, are there any other places that's used or can we get rid of it? ping @jboyd01

**Comment by jboyd01** [comment]
slacked with Mike, issue is gofmt (1.10 vs 1.11 I believe).  The custom resource under hack is just for testing.

**Comment by damemi** [comment]
Updated gofmt and pushed (thanks Jay!) hopefully this is good now

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -46,16 +42,6 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
-	dynamicClient, err := dynamic.NewForConfig(ctx.KubeConfig)
-	if err != nil {
-		return err
-	}
-
-	v1helpers.EnsureOperatorConfigExists(
```
Comment (jboyd01) [inline pkg/operator/starter.go:54]: @damemi as I review this again after seeing the e2e error, I'm thinking removing the EnsureOperatorConfigExists()  without doing something else to get the yaml processed isn't going to work.  If this call is removed, what's going to apply or create this yaml?  The CVO has no awareness of this content under bindata (ie its not within /manifests).

/hold

Comment (damemi) [reply pkg/operator/starter.go:54]: @jboyd01 it is under manifests, part of this is moving that out of bindata to there https://github.com/openshift/cluster-svcat-apiserver-operator/pull/47/files#diff-72907faadd83d28353576644ccbb29c2

Comment (jboyd01) [reply pkg/operator/starter.go:54]: Thanks, I see it now

**Pull request merged**